<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1152</width>
    <height>720</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>3DViewer</string>
  </property>
  <property name="accessibleName">
   <string/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>0</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Open OBJ</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>1050</x>
      <y>0</y>
      <width>101</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>35</y>
      <width>201</width>
      <height>361</height>
     </rect>
    </property>
    <property name="title">
     <string>Moving</string>
    </property>
    <widget class="QLabel" name="labelMoveX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>24</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Move X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollMoveX">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>51</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoveY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>70</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Move Y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollMoveY">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>97</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollMoveZ">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>150</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-200</number>
     </property>
     <property name="maximum">
      <number>200</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelMoveZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>123</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Move Z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollRotationZ">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>296</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelRotationZ">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>269</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rotation z</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelRotationY">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>216</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rotation y</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollRotationY">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>243</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QScrollBar" name="scrollRotationX">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>197</y>
       <width>200</width>
       <height>15</height>
      </rect>
     </property>
     <property name="minimum">
      <number>-180</number>
     </property>
     <property name="maximum">
      <number>180</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="labelRotationX">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>170</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Rotation x</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="labelScale">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>313</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Scale</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_2">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>330</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>-</string>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_3">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>330</y>
       <width>50</width>
       <height>25</height>
      </rect>
     </property>
     <property name="text">
      <string>+</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>400</y>
      <width>201</width>
      <height>241</height>
     </rect>
    </property>
    <property name="title">
     <string>Settings</string>
    </property>
    <widget class="QLabel" name="labelDots">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Dots</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonColorDots">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>45</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Color dots</string>
     </property>
    </widget>
    <widget class="QComboBox" name="boxTypeDots">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>20</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>None</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Circle</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Square</string>
      </property>
     </item>
    </widget>
    <widget class="QScrollBar" name="scrollBarThicknessDots">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>80</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonColorLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>130</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Color line</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>105</y>
       <width>60</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Line</string>
     </property>
    </widget>
    <widget class="QComboBox" name="boxTypeLine">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>105</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Solid</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>dashed</string>
      </property>
     </item>
    </widget>
    <widget class="QScrollBar" name="scrollBarThicknessLine">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>165</y>
       <width>200</width>
       <height>16</height>
      </rect>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonColorBack">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>210</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Color back</string>
     </property>
    </widget>
    <widget class="QLabel" name="labelProjection">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>185</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Projection</string>
     </property>
    </widget>
    <widget class="QComboBox" name="boxTypeProjection">
     <property name="geometry">
      <rect>
       <x>102</x>
       <y>185</y>
       <width>103</width>
       <height>32</height>
      </rect>
     </property>
     <item>
      <property name="text">
       <string>Central</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Parallel</string>
      </property>
     </item>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>649</y>
      <width>201</width>
      <height>70</height>
     </rect>
    </property>
    <property name="title">
     <string>Save</string>
    </property>
    <widget class="QPushButton" name="buttonGif">
     <property name="geometry">
      <rect>
       <x>119</x>
       <y>40</y>
       <width>81</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>gif</string>
     </property>
    </widget>
    <widget class="QPushButton" name="buttonScreen">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>40</y>
       <width>111</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>Screen</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonBMP">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>bmp</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButtonJPEG">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>jpeg</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
